package.json

```diff
@@ -58,7 +58,6 @@
     "lint:angular": "eslint lib/msal-angular/src --ext .ts",
     "lint:browser": "eslint lib/msal-browser/src --ext .ts",
     "lint:common": "eslint lib/msal-common/src --ext .ts",
-    "lint:core": "eslint lib/msal-core/src --ext .ts",
```

Comment:
Rather than removing, let's edit the path. This step will still be run in CI if we need to make changes to the lib

Reply:
I'd like for us to discuss the merit of having things in the "maintenance" directory now that we have an LTS branch. Should we be making changes to unsupported maintenance libs in the dev branch or on LTS? I would argue that these changes should be made in LTS because they are out of active development.

Reply:
I have no strong opinion but I agree these should exist on one branch or the other, not both